bitnami/harbor/values.yaml

```diff
@@ -2914,9 +2914,20 @@ trivy:
   ## @param trivy.skipUpdate The flag to disable Trivy DB downloads from GitHub
   ## You might want to set the value of this flag to `true` in test or CI/CD environments to avoid GitHub rate limiting issues.
   ## If the value is set to `true` you have to manually download the `trivy.db` file and mount it in the
-  ## `/home/scanner/.cache/trivy/db/trivy.db` path.
+  ## `/bitnami/harbor-adapter-trivy/.cache/trivy/db/trivy.db` path.
+  ## ref: https://trivy.dev/dev/docs/configuration/db/
   ##
   skipUpdate: false
+  ## @param trivy.skipJavaDbUpdate The flag to disable Trivy JAVA DB downloads.
+  ## You might want to set the value of this flag to `true` in test or CI/CD environments to avoid GitHub rate limiting issues.
+  ## If the value is set to `true` you have to manually download the `trivy-java.db` file and mount it in the
+  ## `/bitnami/harbor-adapter-trivy/.cache/trivy/java-db/trivy-java.db` path.
+  ##
+  skipJavaDbUpdate: false
+  ## @param trivy.dbRepository OCI repositor(ies) to retrieve the trivy vulnerability database from
+  dbRepository: "mirror.gcr.io/aquasec/trivy-db,ghcr.io/aquasecurity/trivy-db"
+  ## @param trivy.javaDbRepository OCI repositor(ies) to retrieve the Java trivy vulnerability database from
+  javaDbRepository: "ghcr.io/aquasecurity/trivy-java-db"
```

Comment:
We shouldn't set these parameter by default and rely on Trivy's default repositories. This way, if Trivy changes the default repos we don't need to track them.